server/services/interventionsService.test.ts

```diff
@@ -1069,6 +1072,43 @@ pactWith({ consumer: 'Interventions UI', provider: 'Interventions Service' }, pr
       expect(await interventionsService.getSentReferrals(token)).toEqual([sentReferral, sentReferral])
     })
   })
+
+  describe('getInterventions', () => {
+    it('returns a list of all interventions', async () => {
+      const intervention = {
+        id: '15237ae5-a017-4de6-a033-abf350f14d99',
+        title: 'Better solutions (anger management)',
+        description:
+          'To provide service users with key tools and strategies to address issues of anger management and temper control and explore the link between thoughts, emotions and behaviour. It provides the opportunity for service users to practice these strategies in a safe and closed environment.',
+        pccRegions: [
+          { id: 'cheshire', name: 'Cheshire' },
+          { id: 'cumbria', name: 'Cumbria' },
+          { id: 'lancashire', name: 'Lancashire' },
+          { id: 'merseyside', name: 'Merseyside' },
+        ],
+        serviceCategory: serviceCategoryFactory.build(),
+        serviceProvider: serviceProviderFactory.build(),
+        eligibility: eligibilityFactory.allAdults().build(),
+      }
+
+      await provider.addInteraction({
+        state: 'There are some interventions',
+        uponReceiving: 'a request for all interventions',
```

Comment:
Would "searching for interventions without filters" make sense? It explains the reason we're calling this endpoint from the UI's perspective.